logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/keypackage/KeyPackageManager.kt

```diff
@@ -55,7 +56,14 @@ internal class KeyPackageManagerImpl(
 
     private suspend fun refillKeyPackagesIfNeeded() {
         keyPackageRepository.value.lastKeyPackageCountCheck().flatMap { timestamp ->
-            if (Clock.System.now().minus(timestamp) > KEY_PACKAGE_COUNT_CHECK_DURATION) {
+            val forceRefill = keyPackageCountUseCase.value().let {
```

Comment:
Shouldn't we pass `fromAPI` false here? 

```suggestion
            val forceRefill = keyPackageCountUseCase.value(fromAPI = false).let {
```